controller/src/test/java/org/jboss/as/controller/persistence/ConfigurationFileUnitTestCase.java

```diff
@@ -0,0 +1,123 @@
+package org.jboss.as.controller.persistence;
```

Comment:
Please add the standard ASL copyright header. Thanks.

Reply:
@bstansberry did you mean the LGPL header?

Reply:
@jmesnil @adamecp I meant the ASL header. All new files in WildFly Core should use the ASL header, not the LGPL one.